geometry_msgs/PointStamped attention_point
---
geometry_msgs/Pose2D goal_pose
